src/mavedb/lib/authentication.py

```diff
@@ -86,15 +78,15 @@ def verify_jwt(token: str) -> dict:
 
 
 async def get_access_token(
-        # access_token_query: str = Security(access_token_query),
-        access_token_header: Optional[str] = Security(access_token_header),
-        access_token_cookie: Optional[str] = Security(access_token_cookie),
+    # access_token_query: str = Security(access_token_query),
```

Comment:
why this commented out line?

Reply:
We can certainly get rid of it, but the line isn't changed in the current PR -- aside from the indentation, since Black has been run inconsistently in the past.

Reply:
ah, missed that